etc/omero.properties

```diff
@@ -826,6 +826,9 @@ omero.client.ui.menu.dropdown.everyone=All Members
 # Flag to show/hide all users.
 omero.client.ui.menu.dropdown.everyone.enabled=true
 
+# Client viewers interpolate pixels by default.
+omero.client.interpolate_pixels=true
+
```

Comment:
As this is viewer specific, could we keep naming convention and rename it to omero.client.viewer.interpolate_pixels